### ISSUE TITLE
Remove git-spy from the packer

### ISSUE THREAD

**Issue by shawninder** [vigour-io/packer-server#31]
There is a maximum number of webhooks (20) we can wet on an organization. Will will bust that limit pretty soon at this rate. I think the solution is to stop having one git-spy per packer, but rather having a single git-spy to which many packers can subscribe.

**Comment by shawninder** [comment]
see https://github.com/vigour-io/git-spy/issues/6

**Comment by ninjatux** [comment]
@shawninder this is done right?

**Comment by shawninder** [comment]
Not exactly. It's done the fast way, but not the correct way. To make this 100% correct, I'm going to need a usable hub. So this is kind of on hold for now...
